setup.py

```diff
@@ -20,10 +23,24 @@ def get_long_desc():
     desc += open('CHANGES').read()
     return desc
 
+# foolproof way of single-sourcing version as per
+# http://stackoverflow.com/a/24517154/112943
```

Comment:
While it doesn't use the exact code, I feel like it still uses the same general idea (separate file and reading it without importing it, as opposed to using `setuptools` or trying to import the file somehow). If you still feel this comment does not apply, I don't mind removing it.